CITATION.cff

```diff
@@ -0,0 +1,178 @@
+cff-version: 1.1.0
+title: "libCEED: the CEED Library"
+version: 0.9.0
+license:  BSD-2-Clause
+message: "Please cite the following works when using this software."
+authors:
+  - family-names: Brown
+    given-names: Jed
+    orcid: 0000-0002-9945-0639
+  - family-names: Abdelfattah
+    given-names: Ahmad
+    orcid: 0000-0001-5054-4784
+  - family-names: Barra
+    given-names: Valeria
+    orcid: 0000-0003-1129-2056
+  - family-names: Beams
+    given-names: Natalie
+    orcid: 0000-0001-6060-4082
+  - family-names: Camier
+    given-names: Jean Sylvain
+    orcid: 0000-0003-2421-1999
+  - family-names: Dobrev
+    given-names: Veselin
+    orcid: 0000-0003-1793-5622
+  - family-names: Dudouit
+    given-names: Yohann
+    orcid: 0000-0001-5831-561X
+  - family-names: Ghaffari
+    given-names: Leila
+    orcid: 0000-0002-0965-214X
+  - family-names: Kolev
+    given-names: Tzanio
+    orcid: 0000-0002-2810-3090
+  - family-names: Medina
+    given-names: David
+  - family-names: Pazner
+    given-names: Will
+    orcid: 0000-0003-4885-2934
+  - family-names: Ratnayaka
+    given-names: Thilina
+    orcid: 0000-0001-6102-6560
+  - family-names: Thompson
+    given-names: Jeremy L
+    orcid: 0000-0003-2980-0899
+  - family-names: Tomov
+    given-names: Stanimire
+    orcid: 0000-0002-5937-7959
+date-released: 2021-07-07
+
```

Comment:
```suggestion
doi: 10.5281/zenodo.5080234

```
This should work according to this example https://github.com/citation-file-format/citation-file-format

Reply:
I got that one. The "view citation file" button still doesn't work for me, and I'm not sure exactly why, but the generation of the APA and Bibtex is working correctly.

Reply:
Yeah, I was just hoping to add the DOI because the current approach seems really poorly thought out (no URL, no publisher, no DOI, just the software name).

Reply:
Yea. I swept that change up into my other changes to the base file.

Reply:
Looking at the generation citations, it seems that the DOI is the most important of the three - it is the only one present in the APA format